### PR TITLE
Refs #31910 - Remove katello-agent from ping model

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by jturel** [comment]
@chris1984 we still need to show it if katello_agent is enabled which we now have a handy method to check that with!

**Comment by jturel** [comment]
Before you push the next update you may wanna run the ping model test locally cause good chance it might break something

File: app/models/katello/ping.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def pulpcore_enabled # for downstream 6.9, remove in 6.10
 
       def services(capsule_id = nil)
         proxy = fetch_proxy(capsule_id)
-        services = [:candlepin, :candlepin_auth, :foreman_tasks, :katello_events, :candlepin_events, :katello_agent]
+        services = [:candlepin, :candlepin_auth, :foreman_tasks, :katello_events, :candlepin_events]
         services += [:pulp3] if proxy&.pulp3_enabled? && pulpcore_enabled
```
Comment (ehelms) [inline app/models/katello/ping.rb:18]: Could we add a line like this one for katello_agent inclusion?

Comment (chris1984) [reply app/models/katello/ping.rb:18]: yeah, will update it, i thought you guys wanted it totally removed, will add it back if the agent is present.

**Comment by jturel** [comment]
Also need to see how hammer behaves when katello_agent isn't enabled. Does it show a blank field for it or something?

**Comment by chris1984** [comment]
> Also need to see how hammer behaves when katello_agent isn't enabled. Does it show a blank field for it or something?

+1 will look into that as well.

**Comment by chris1984** [comment]
> Before you push the next update you may wanna run the ping model test locally cause good chance it might break something

Tests pass, will update them as well since they are not testing much:
```bash
Finished in 6.020414s, 4.1525 runs/s, 11.1288 assertions/s.

25 runs, 67 assertions, 0 failures, 0 errors, 0 skips
```

**Comment by jturel** [comment]
One more thing - need to disable the AgentEventReceiver https://github.com/Katello/katello/blob/628ee7ed2a40ec1ea33c1d84bd940035ad34bdec/lib/katello/engine.rb#L158
.. only register the service if katello agent is enabled

**Review by jturel — APPROVED** [review]
APJ

Without agent:
```
{
  "results": {
    "foreman": {
      "database": {
        "active": true,
        "duration_ms": "0"
      }
    },
    "katello": {
      "services": {
        "candlepin": {
          "status": "ok",
          "duration_ms": "15"
        },
        "candlepin_auth": {
          "status": "ok",
          "duration_ms": "14"
        },
        "foreman_tasks": {
          "status": "ok",
          "duration_ms": "2"
        },
        "katello_events": {
          "status": "ok",
          "message": "0 Processed, 0 Failed",
          "duration_ms": "0"
        },
        "candlepin_events": {
          "status": "ok",
          "message": "0 Processed, 0 Failed",
          "duration_ms": "0"
        },
        "pulp3": {
          "status": "ok",
          "duration_ms": "40"
        }
      },
      "status": "ok"
    }
  }
}

```

with agent:
```
{
  "results": {
    "foreman": {
      "database": {
        "active": true,
        "duration_ms": "0"
      }
    },
    "katello": {
      "services": {
        "candlepin": {
          "status": "ok",
          "duration_ms": "15"
        },
        "candlepin_auth": {
          "status": "ok",
          "duration_ms": "35"
        },
        "foreman_tasks": {
          "status": "ok",
          "duration_ms": "3"
        },
        "katello_events": {
          "status": "ok",
          "message": "0 Processed, 0 Failed",
          "duration_ms": "0"
        },
        "candlepin_events": {
          "status": "ok",
          "message": "0 Processed, 0 Failed",
          "duration_ms": "0"
        },
        "pulp3": {
          "status": "ok",
          "duration_ms": "45"
        },
        "katello_agent": {
          "status": "ok",
          "message": "0 Processed, 0 Failed",
          "duration_ms": "0"
        }
      },
      "status": "ok"
    }
  }
}

```

File: app/models/katello/ping.rb [hunk 2]
```diff
@@ -75,6 +76,7 @@ def ping_katello_agent(result)
           status = Katello::EventDaemon::Services::AgentEventReceiver.status(refresh: false)
           event_daemon_status(status, result)
         end
+
```
[the remark this answers is not in the document]
Comment (chris1984) [reply app/models/katello/ping.rb:79]: working on pushing up an update

**Pull request merged**